### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by mewmew** [luxengine/lux#22]
From the readme

> Open source solution for UI (preferably html).

One option would be to look into [electron](https://github.com/electron/electron) (http://electron.atom.io/), and see if there exists good bindings for it in Go.

**Comment by hydroflame** [comment]
Yeah I looked into that and CEF itself. The problem is that they both want multiple processes and CEF even wants multiple applications and a very specific folder structure to be used on OSX. Making it very hard to operate cross platform. I'm thinking about maybe having someone rip out the offline renderer from CEF and have just that on it's own.

Ideally we'd get a pure go offline renderer though, a man can dream

**Comment by hydroflame** [comment]
steam might provide something :)
